projects/packages/forms/src/blocks/contact-form/editor.scss

```diff
@@ -380,6 +380,13 @@
 		outline: none;
 		position: relative;
 		pointer-events: none;
+		border: none;
+
+		/* Some classic themes set the min-widths, which gives an elliptical shape to the input. */
+		min-width: auto;
+		width: 1em;
+		height: 1em;
+		box-shadow: none;
```

Comment:
This is to avoid some really weird 3d effects on classic themes, but testing in trunk they're not there. 

I'll look into a better fix.

Reply:
Uh, here's the main feature branch so I think it's something we do need to address:

<img width="443" alt="Screenshot 2025-05-22 at 1 01 38 pm" src="https://github.com/user-attachments/assets/259ca0a6-a60a-4482-b134-e6db579575d1" />

I'll leave these styles then here. 

What's happening is that [we added width/height of `1em` so the inputs scale with the font size](https://github.com/Automattic/jetpack/pull/42890/files#diff-b032342e36da30ca1c7a18a770aac75a9d52bdec892fe09e3e047c3a8ca73b69R391-R395).

We also removed the border and height overrides: https://github.com/Automattic/jetpack/pull/42890/files#diff-b032342e36da30ca1c7a18a770aac75a9d52bdec892fe09e3e047c3a8ca73b69L368

So I think we need to keep these new declarations here to compensate.

Reply:
Not removed, it was moved to here to lower the specificity:
https://github.com/Automattic/jetpack/blob/dd7773c0863eb55a5ae08ffb1a1764f7714d3a00/projects/packages/forms/src/blocks/contact-form/editor.scss#L388

That looks like a mistake. Maybe these rules should all have `!important` if the border/outline is never to be shown. 🤔

Reply:
Probably the same with `appearance: none` as well, if that's overridden by a theme's css then border/outline being set to `none` won't have any effect.

Reply:
>  Maybe these rules should all have !important if the border/outline is never to be shown.

That the border/outline is never to be shown, was my assumption after testing in trunk. At least the contact form CSS is opinionated on the matter. 😄 

What's your recommendation here? Should I move the following from `:where(.jetpack-option__type) {` to `.jetpack-option__type.jetpack-option__type`:

```css
height: 1em;
width: 1em;
border: none;
```

And give them an `!important`?

Reply:
Yep, I think `!important` might be the best choice for those, and also the `outline` and `appearance`. Just need to re-test to make sure it doesn't affect the application of global styles and block styles in any way.

Reply:
My guess is that as long as the `font-size` rule is still at 0-1-0 specificity it should be ok.

Reply:
Done in [a5a6b2b](https://github.com/Automattic/jetpack/pull/43531/commits/a5a6b2b1a11c2602bbd21537033524b75ca82e10)

Looks pretty consistent in my testing. 

The only thing I noticed was the mismatch in label/input colors on the option block, but I think you're looking at that.